desktop-widgets/tab-widgets/TabDiveEquipment.cpp

```diff
@@ -81,7 +81,8 @@ TabDiveEquipment::TabDiveEquipment(QWidget *parent) : TabBase(parent),
 	for (int i = 0; i < CylindersModel::COLUMNS; i++) {
 		if (ignoreHiddenFlag(i))
 			continue;
-		bool checked = s.value(QString("column%1_hidden").arg(i)).toBool();
+		auto setting = s.value(QString("column%1_hidden").arg(i));
+		bool checked = !setting.isValid() && i == CylindersModel::SENSORS ? true : setting.toBool();
```

Comment:
Careful: The CylindersModel code is a complete disaster, because it is used in two rather different places: in the planner and the equipment tab.

I could imagine that the column should not be shown in the equipment tab at all.

Reply:
This code didn't seem to be used in the planner and the column was always visible there, so I made it always hidden in the planner.

Still for the non-planner case, is this how we want to have it? Keep it hidden by default and people who might need it can discover it later? If so, should i make it more generic and possibly hide some other column by default?

Reply:
Decided to keep it hidden by default since I didn't get a reply to this. Also made it a bit more generic so it's easier to add new column as hidden by default if that need comes up.